opl/generators/edge_inventory_ingress_template.json.j2

```diff
@@ -634,14 +634,6 @@
             "selinux_current_mode": "enforcing",
             "infrastructure_vendor": "kvm",
             "rpm_ostree_deployments": [
-                {
-                    "id": "rhel-edge-ec3c003da4eafaa971b528b3383d8caff688a110e53af71a85e666cf60b4ed20.0",
-                    "booted": true,
-                    "origin": "rhel-edge:rhel/8/x86_64/edge",
-                    "osname": "rhel-edge",
-                    "pinned": false,
-                    "checksum": "{{ os_tree_commit }}"
```

Comment:
Hello @jhutar . I removed this because i was getting these errors in inventory logs
```
app.exceptions.ValidationException: {'system_profile': ["System profile does not conform to schema.\n't1upkewfz3dtiqm7l14orb1aijtcipwnqjt70s1b9p6rs2ptnb4ar60xuj0ab1g7' does not match '^[a-fA-F0-9]{64}$'\n\nFailed validating 'pattern' in schema['properties']['rpm_ostree_deployments']['items']['properties']['checksum']:\n    {'description': 'The checksum / commit of the deployment
```
I was able to overcome these errors with these values.